app/errors.py

```diff
@@ -0,0 +1,14 @@
+class VaccineError(Exception):
```

Comment:
The code looks good overall! Just a minor suggestion: consider adding docstrings to your custom exception classes to provide a brief explanation of their purpose. This will make the code easier to understand for other developers 😊